src/Components/v2/ArtworkFilter/ArtworkFilterContext.tsx

```diff
@@ -10,12 +10,12 @@ import { rangeToTuple } from "./Utils/rangeToTuple"
  */
 export const initialArtworkFilterState = {
   attribution_class: [],
-  height: "*-*",
+  // height: "*-*",
```

Comment:
Not 100% sure, but these seemed to be defaults in one request but not another. This, combined with that additional `artworks_connection` load means the initial render of an artist page (unfiltered), takes one request.

Reply:
This might have an impact on other pages, let me see...